src/fn/upload-custom-translations.js

```diff
@@ -10,6 +12,7 @@ module.exports = (projectDir, couchUrl) => {
 
   const dir = `${projectDir}/translations`;
   const db = pouch(couchUrl);
+  const instanceUrl = couchUrl.replace(/\/medic$/, '');
```

Comment:
This doesn't work when the db name isn't `medic`. In api/src/environment.js we do this...

```js
  const parsedUrl = url.parse(couchUrl.replace(/\/$/, ''));
  const instanceUrl = couchUrl.slice(0, couchUrl.lastIndexOf('/')),
```

Maybe we should have an `environment` module for medic-conf too?

Reply:
The whole of medic-conf doesn't work with db names that aren't `medic`:
https://github.com/medic/medic-conf/blob/master/src/lib/main.js#L171

I thought that would be part of a separate issue. Would you like me to include an overall update here?

Reply:
Oh dear... Please create a separate issue to cover that.